packages/server/postgres/queries/src/getPollOptionsByPollIdsQuery.sql

```diff
@@ -0,0 +1,6 @@
+/*
+  @name getPollOptionsByPollIdsQuery
+  @param pollIds -> (...)
+*/
+SELECT * FROM "PollOption"
```

Comment:
I keep wondering if we'll ever query a PollOption without querying a Poll, or vice versa.
In NoSQL, we'd simply have all the options live in the same document as the poll.
In SQL, it makes sense to properly normalize them. 

Is it worthwhile writing 1 query that fetches a poll and all its options? that means we could get everything we need with a single trip to the database.